package.json

```diff
@@ -57,7 +57,7 @@
     "@types/styled-components": "^5.1.25",
     "ajv": "^8.11.0",
     "concurrently": "^7.2.2",
-    "electron": "^14.2.9",
+    "electron": "^20.0.2",
```

Comment:
I tested this earlier and everything seemed ok. Per @vigneshshanmugam's comment I went back and checked the Test functionality again and now I'm seeing an error like:

```
[2] Error: 
[2] Something went wrong installing the "sharp" module
[2] 
[2] Cannot find module '../build/Release/sharp-darwin-arm64v8.node'
```

The Test feature is broken on this branch.

Reply:
It might be that you have installed the modules while you were on node 14? Node 14 doesn't support Apple Silicon so when installing sharp(which is a native module), it downloads darwin-x64. But the latest version of electron uses node 16 and it can recognize if the host machine is Apple Silicon and it expects to find the arm build of sharp.

Try to run:
1. nvm use
2. npm ci

and run the test feature again, you shouldn't see the warning again.

Reply:
You are probably right. I likely tested in a shell with the proper node version firstly, and the second time failed to use the right version. I'll test again soon.

Reply:
Yeah you were right, I must've not been on the appropriate node version.